Formula/drafter.rb

```diff
@@ -0,0 +1,26 @@
+class Drafter < Formula
+  desc "Snow Crash parser harness"
+  homepage "https://apiblueprint.org"
+  url "https://github.com/apiaryio/drafter/releases/download/v3.2.7/drafter-v3.2.7.tar.gz"
+  sha256 "a2b7061e2524804f153ac2e80f6367ae65dfcd367f4ee406eddecc6303f7f7ef"
+  head "https://github.com/apiaryio/drafter.git"
+
+  def install
+    system "./configure", "--shared"
+    system "make", "drafter"
+    bin.install "bin/drafter"
+    (include + "drafter").install "src/drafter.h"
+    lib.install "build/out/Release/libdrafter.dylib"
```

Comment:
Please report to upstream the fact that `make install` does not install shared library and header, as it should.

Reply:
I found this one already open: https://github.com/apiaryio/drafter/issues/426 is that specific enough?